chat-commands.js

```diff
@@ -2443,14 +2443,15 @@ exports.commands = {
 
 	hotpatch: function (target, room, user) {
 		if (!target) return this.parse('/help hotpatch');
-		if (!this.can('hotpatch')) return false;
-		if (Monitor.hotpatchLock) return this.errorReply("Hotpatch is currently been disabled. (" + Monitor.hotpatchLock + ")");
-
-		Rooms.global.notifyRooms(['development', 'staff', 'upperstaff'], `|c|${user.getIdentity()}|/log ${user.name} used /hotpatch ${target}`);
+		if (!this.can('hotpatch')) return;
+		const rooms = ['development', 'staff', 'upperstaff'];
+		const prefix = `|c|${user.getIdentity()}|/log `;
+		const hotpatchLock = Monitor.hotpatchLock;
```

Comment:
Defining a variable seems overkill to chop off literally `Monitor.` - at least call it `lock`.